challenge.md

```diff
@@ -33,18 +33,24 @@ Open up this file in your text editor, and write your answers below (where indic
 
 <!-- Answer Starts Here -->
 
+Absolute path - the path begins with a forward slash.
+
 <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
 <!-- Answer Starts Here -->
 
+We would use an absolute path to refer to the font, since the font doesn't live specifically within our project.
+
```

Comment:
This is true. But in fact, we don't actually have a choice in the matter - when we refer to something on the web, we do so by its URL, and a URL is an absolute path.